addons/sys_gestures/fnc_stopGesture.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Handles stopping the radio gesture
+ *
+ * Arguments:
+ * 0: Unit to stop gesture <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [acre_player] call acre_sys_gesture_fnc_stopGesture
+ *
+ * Public: No
+ */
+
+params ["_unit"];
+
+if (_unit getVariable [QGVAR(onRadio), false]) then {
```

Comment:
`exitWith` to reduce nesting? just my preference.

Reply:
Not mine, but added anyway 💯